docs/scos/dev/feature-walkthroughs/202204.0/click-and-collect-feature-walkthrough/service-point-cart-subdomain-walkthrough.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Click and Collect feature Service Point Cart subdomain walkthrough
+last_updated: Nov 02, 2023
+description: TODO
+template: concept-topic-template
+---
+
+# Service Point Cart
+
+Provides support for service points validation and replacement in a cart.
```

Comment:
Enables validation of service points and replacement of line items in a shopping cart.